packages/xo-server/src/xapi/index.js

```diff
@@ -737,6 +737,9 @@ export default class Xapi extends XapiBase {
         this._deleteVm(snapshot)
       )::ignoreErrors(),
 
+      vm.suspend_VDI !== NULL_REF &&
```

Comment:
Only if the power_state is `suspended` (doc: *Only has meaning if VM is currently suspended*).